docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -209,8 +213,26 @@ Release
       git tag --annotate "${TAG}" "${BRANCH}" -m "Tag version ${TAG}"
       git push origin "${TAG}"
 
+#. Go to `Jenkins`_ and force a build on the release branch to test the latest commit.
+
+   Currently, jobs cannot be created for git tags so the latest commit must be tested instead.
+   This must be the same commit as the tag.
+   The git commit that was used can be seen on the summary page for any build.
+   To test this commit, force a build of the ``__main_multijob`` job and any other jobs which are not triggered by this.
+
+   If :issue:`3917` has not been resolved, then a build of the ``_build_vagrant_basebox_for_flocker_tutorial`` job must also be triggered.
```

Comment:
I'm putting https://github.com/ClusterHQ/flocker/compare/kill-vagrant-FLOC-3916-FLOC-3917 up for review soon, so these instructions can be removed.